e2e/pageobjects/ide/Editor.ts

```diff
@@ -13,44 +13,74 @@ import { DriverHelper } from '../../utils/DriverHelper';
 import { CLASSES } from '../../inversify.types';
 import { TestConstants } from '../../TestConstants';
 import { By, Key } from 'selenium-webdriver';
+import { error } from 'selenium-webdriver';
```

Comment:
This can be appended to the upper row as ` import { By, Key, error } from 'selenium-webdriver';  `